### ISSUE TITLE
No return to source app during login after password change

### ISSUE THREAD

**Issue by kanstantsin** [VKCOM/vk-ios-sdk#173]
Hi!
There's an issue with source app callback when the password has been changed.
Steps:
1. Change password on web (vk.com).
2. Try to authorize with VK iOS SDK ([VKSdk authorize:]).
3. VK app opens and asks to enter login and password.
4. Enter login and new password.

Result:
VK app stays open.

Expectation:
Source app is opened with new token.

**Comment by Dreddik** [comment]
Is your url-scheme correct? Please check this out

**Comment by kanstantsin** [comment]
Yes, it's correct. OAuth works fine for all cases except of this one.

**Comment by rostislawk** [comment]
Same problem with registration new user. VK app stays open and doesn't return to source app
